stetho/src/main/java/com/facebook/stetho/inspector/elements/DOMProvider.java

```diff
@@ -23,6 +23,8 @@
 
   public void setInspectModeEnabled(boolean enabled);
 
+  void setAttributesAsText(Object element, String text);
```

Comment:
Please match the style of the surrounding code by putting 'public' in front of the method.